LoopBackTests/server/index.js

```diff
@@ -144,14 +141,14 @@ app.model('Customer', {
         foreignKey: "userId"
       }
     }
-  },
-  dataSource: 'Memory'
-});
+  });
+
+app.model(Customer, { dataSource: 'Memory' });
+
+app.dataSource('mail', { connector: 'mail' });
```

Comment:
Since we are not calling `autoAttach`, the `mail` datasource should not be needed at all now.